lib/hammer_cli_katello/repository.rb

```diff
@@ -70,6 +70,10 @@ class InfoCommand < HammerCLIKatello::InfoCommand
           field :docker_manifest_total, _("Docker Manifests"), Fields::Field, :hide_blank => true
           field :docker_tag_total, _("Docker Tags"), Fields::Field, :hide_blank => true
         end
+        collection :ostree_branches, _("RPM OS Tree Branches"),
```

Comment:
Wondering if I should just hide this field and show content counts like the other content types. Thoughts?